### PR TITLE
Node consistency 31 40

### PR DESCRIPTION
**Pull request by martinstacey**
### Purpose

The following PR addresses nodes' names and descriptions to make them more descriptive for beginners.

**31) NODE:  Vector.Dot**
-FYI: is inside Protogeometry.dll should be changed by Autodesk CHANGE SUGGESTED: i/o naming and descriptions

**32) NODE:  CoordinateSystem.Transform**
-FYI: is inside Protogeometry.dll should be changed by Autodesk  CHANGE SUGGESTED: Seems to not function at all?

**33) NODE:  CoordinateSystem.Scale1D**
-FYI: is inside Protogeometry.dll should be changed by Autodesk  CHANGE SUGGESTED:  Description needs work

**34) NODE:  CoordinateSystem.Mirror**
-CHANGES: No changes

**35) NODE:  CoordinateSystem.IsEqualTo**
-FYI: is inside Protogeometry.dll should be changed by Autodesk  CHANGE SUGGESTED: Revise input name, add descriptions

**36) NODE:  BoundingBox.IsEmpty**
-FYI: is inside Protogeometry.dll should be changed by Autodesk  CHANGE SUGGESTEDI am not too sure how to get this to return "TRUE". Node is odd.

**37) NODE:  BoundingBox.Intersects**
-FYI: is inside Protogeometry.dll should be changed by Autodesk  CHANGE SUGGESTED: Input/Output cleanup for name and decription.

**38) NODE: Watch Image**
-CHANGES: 
Input Description: Image for visualization

**39) NODE: Watch 3D**
-CHANGES: No changes

**40) NODE: Color Red**
-CHANGES: 
Output= red to int.
Output Description: Red value for RGB color model, int between 0 and 255 inclusive.

image

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers
@SHKnudsen
@Amoursol 
@QilongTang 

### FYIs
-	Nodes 31-33, 35-37: inside Protogeometry.dll should be changed by Autodesk.

### CODE REVIEW

**Comment by Amoursol** [comment]
@martinstacey Please provide improvement suggestions for all the nodes that are locked into ProtoGeometry behind ADSK VPN.

**Review by QilongTang — APPROVED** [review]

File: src/Libraries/CoreNodes/Color.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public class Color
         /// <summary>
         ///     Find the red component of a color, 0 to 255.
         /// </summary>
-        /// <returns name="red">int between 0 and 255 inclusive.</returns>
+        /// <returns name="integer">Red value for RGB color model, int between 0 and 255 inclusive.</returns>
```
Comment (aparajit-pratap) [inline src/Libraries/CoreNodes/Color.cs:22]: I think we should just say `int`. A lot of nodes just have `int`.

Comment (martinstacey) [reply src/Libraries/CoreNodes/Color.cs]: addressed in b310d8f

**Comment by martinstacey** [comment]
> @martinstacey Please provide improvement suggestions for all the nodes that are locked into ProtoGeometry behind ADSK VPN.

@Amoursol 

Suggestions for nodes in protogeometry:

**Vector.Dot:**
Input Name: vector1
Input description vector1 : a vector
Input Name:vec -> vector2
Input description vector2 : a second vector
Output description double: resulting dot product

**CoordinateSystem.Transform:**
This node doesn't seem to work.

**CoordinateSystem.Scale1D:**
Node Description: Scale in one dimension by using a base point, a starting point (from), and an end point (to). The scaling axis is defined by the line between base point and start point.

**CoordinateSystem.IsEqualTo:**
Input Name: coordinateSystem
Input description coordinateSystem1: a coordinate system
Input Name: coordinateSystem2
Input Description coordinateSystem2: another coordinate system

**BoudingBox.IsEmpty:**
This node doesn't seem to work, does it determine if there are geometries inside the bounding box?

**BoudingBox.Intersects:**
Input Name: boundingBox1
Input description boundingBox1: a bounding box
Input Name: boundingBox2
Input Description boundingBox2: another bounding box
Output Description bool: do bounding boxes intersect

**Comment by Amoursol** [comment]
JIRA task located here: https://jira.autodesk.com/browse/DYN-3146

**Review by Amoursol — APPROVED** [review]

File: src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx [hunk 2]
```diff
@@ -388,7 +388,7 @@
     <value>False block</value>
   </data>
   <data name="PortDataImageToolTip" xml:space="preserve">
-    <value>image</value>
+    <value>Image for visualization</value>
```
Comment (aparajit-pratap) [inline src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx:391]: I think the same change needs to be made to the `Resources.resx` file.

Comment (martinstacey) [reply src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx:391]: Addressed in b589c6c

**Review by aparajit-pratap — APPROVED** [review]

**Pull request merged**